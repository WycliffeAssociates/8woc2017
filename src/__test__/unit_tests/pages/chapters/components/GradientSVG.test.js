/* global it expect describe*/
import React from 'react';
import GradientSVG from '../../../../../js/pages/chapters/components/GradientSVG';
import renderer from 'react-test-renderer';

describe('Chapter card comments', () => {
  it('renders correctly and matches snapshot', () => {
    const tree = renderer
      .create(<GradientSVG />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});
