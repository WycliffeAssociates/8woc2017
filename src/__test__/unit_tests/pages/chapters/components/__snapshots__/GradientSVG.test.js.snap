// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chapter card comments renders correctly and matches snapshot 1`] = `
<svg
  style={
    Object {
      "height": 0,
    }
  }
>
  <defs>
    <linearGradient
      gradientTransform="rotate(undefined)"
      id={undefined}
    >
      <stop
        offset="0%"
        stopColor={undefined}
      />
      <stop
        offset="100%"
        stopColor={undefined}
      />
    </linearGradient>
  </defs>
</svg>
`;
